CHANGELOG.md

```diff
@@ -3,6 +3,103 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.16.2
+
+_Oct 12, 2023_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
```

Comment:
@mui/explore team, please mention if there are some highlights on your side.

Thanks

Reply:
Maybe mention the charts BC in highlights? 🤔 
```suggestion

- 📊 Chart's legend text management has been reworked and contains breaking changes (#10138) 
@alexfauquette  
```